### PR TITLE
add bernoulli to paddle backend

### LINKED ISSUE #19078: bernoulli

### PR DESCRIPTION
**Pull request by mohame54**
close #19078

### CODE REVIEW

**Comment by mohame54** [comment]
I am ready for the review @MuhammadSaeedBatikh

**Comment by mohame54** [comment]
I am ready for the review @MahmoudAshraf97

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]
please use `paddle.bernoulli` directly, don't create a distribution and sample from it
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/bernoulli_en.html#bernoulli

**Comment by mohame54** [comment]
> please use `paddle.bernoulli` directly, don't create a distribution and sample from it https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/bernoulli_en.html#bernoulli

Okay on it

File: ivy/functional/backends/paddle/experimental/random.py [hunk 1]
```diff
@@ -90,6 +91,18 @@ def poisson(
...
+    {
+        "2.5.0 and below": {
+            "cpu": (
+                "float32",
+                "float64",
+            )
+        }
+    },
+    backend_version,
+)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/experimental/random.py:105]:
```suggestion
@with_supported_device_and_dtypes(
    {
        "2.5.0 and above": {
            "cpu": ("float32", "float64"),
            "gpu": ("bfloat16", "float16", "float32", "float64"),
        },
        "2.4.2 and below": {
            "cpu": (
                "float32",
                "float64",
            ),
            "gpu": ("float16", "float32", "float64"),
        },
    },
    backend_version,
)
```

**Review by MahmoudAshraf97 — APPROVED** [review]

**Comment by MahmoudAshraf97** [comment]
ivy-gardener

**Pull request merged**